### DISCUSSION TITLE
94649845-CFF8-4A10-84DC-9FBB1FB49C12

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#127] [General]
# 94649845-CFF8-4A10-84DC-9FBB1FB49C12

NGTS-10 is a binary star system of two K-type main sequence stars about 1,059 light-years away in the constellation Lepus. One of those stars (component A) has a super-Jovian planet that is so close from its star that tidal forces have stretched the planet into a rugby-shaped world, where it completes an orbit once every 18.4 hours at a distance of only 1.89 million km from its host

https://celestia.mobi/addon?item=94649845-CFF8-4A10-84DC-9FBB1FB49C12

**Comment by exocritic5457** [comment]
Any citations for its shape? And less importantly, why does it... look like gum?
